.github/workflows/release.yml

```
@@ -0,0 +1,32 @@
+name: Release
+
+on:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+
+permissions:
+  contents: "read"
+  id-token: "write"
+
+jobs:
+  release:
+    name: Release Starkup to Cloud Storage
+    runs-on: ubuntu-latest
+    steps:
+      - id: "checkout"
+        uses: "actions/checkout@v4"
```

Nit:
We don't really need `"`; and afaik `id` is typically placed after `uses`, `name` 🤔  

```suggestion
      - uses: actions/checkout@v4
        id: checkout
```
same goes for other places